scripts/pirate.py

```diff
@@ -16,6 +17,10 @@ def translate(english):
         for position in matches:
             #Make sure the case is correct.
             uppercase = english[position].lower() != english[position]
+
+            if isinstance(pir, list):
+                pir = random.choice(pir)
```

Comment:
This will not choose a different translation every time you load Cura, but every time this script is ran when we create translations. So for every release basically.